src/main/resources/db/migration/V1_22__delete_incident_updated_status.sql

```diff
@@ -0,0 +1,6 @@
+-- Removes INCIDENT_UPDATED status
+-- See: https://dsdmoj.atlassian.net/wiki/spaces/IR/pages/5589893167/Status+old+and+new
+
+DELETE FROM constant_status
+WHERE code = 'INCIDENT_UPDATED';
```

Comment:
i would have updated the sequence numbers

Reply:
The order is retained, no point in messing with the sequences.